services/frontend-service/src/ui/components/ProductVersion/ProductVersion.tsx

```diff
@@ -132,42 +134,54 @@ export const ProductVersion: React.FC = () => {
     const tagsResponse = useTags();
     const changeEnv = React.useCallback(
         (e: React.ChangeEvent<HTMLSelectElement>) => {
-            const env = splitCombinedGroupName(e.target.value);
             searchParams.set('env', e.target.value);
-            searchParams.set('tag', selectedTag);
             setEnvironment(e.target.value);
             setSearchParams(searchParams);
-            getSummary(selectedTag, env[0], env[1]);
         },
-        [setSearchParams, searchParams, selectedTag]
+        [setSearchParams, searchParams]
     );
     const [displaySummary, setDisplayVersion] = React.useState(false);
     const [showReleaseTrainEnvs, setShowReleaseTrainEnvs] = React.useState(false);
     const handleClose = React.useCallback(() => {
         setShowReleaseTrainEnvs(false);
     }, []);
-    const [showButton, setShowButton] = React.useState(false);
-    React.useEffect(() => {
-        if (localStorage.getItem('testing') !== null) {
-            setShowButton(true);
-        } else {
-            setShowButton(false);
-        }
-    }, []);
     const openDialog = React.useCallback(() => {
         setShowReleaseTrainEnvs(true);
     }, []);
-    const confirmReleaseTrainFunction = React.useCallback((selectedEnvs: string[]) => {
-        selectedEnvs.forEach((env) => {
-            // addAction() call added in DSN-3ZRNTG
-        });
-        return;
-    }, []);
+    const confirmReleaseTrainFunction = React.useCallback(
+        (selectedEnvs: string[]) => {
+            if (teams.length < 1) {
+                selectedEnvs.forEach((env) => {
+                    addAction({
+                        action: {
+                            $case: 'releaseTrain',
+                            releaseTrain: { target: env, commitHash: selectedTag, team: '' },
+                        },
+                    });
+                    return;
+                });
```

Comment:
this `return` looks wrong
```suggestion
                });
                return;
```